### ISSUE TITLE
make install target

### ISSUE THREAD

**Issue by rcoreilly** [KhronosGroup/MoltenVK#531]
would be helpful if the simple provided Makefile included a basic install target, something like this:

```sh
/bin/cp -av Package/Release/MoltenVK/macOS/framework/MoltenVK.framework /Library/Frameworks/
```

**Comment by billhollings** [comment]
Rather than the Makefile...would it be better to include an equivalent script in the runtime package...since any app that depends on that location would need to install it during runtime deployment...not just during dev builds?

I guess the Makefile could call that script to simplify things for devs.

Or should we assume that the app will sort out it's own deployment installation process...and just focus on doing this in the Makefile as suggested?

**Comment by rcoreilly** [comment]
yeah I think an app-level use-case would handle things by itself -- this is for someone who is manually installing and developing vulkan code on their mac, and just wants the stuff in a standard location so they don't have to mess around with additional compiler directives etc.  i.e., the person who is using the Makefile in the first place :)

**Comment by billhollings** [comment]
Fixed in PR #537.
